diff --git a/node_modules/sharp/lib/constructor.js b/node_modules/sharp/lib/constructor.js
index 41f7802..84492b0 100644
--- a/node_modules/sharp/lib/constructor.js
+++ b/node_modules/sharp/lib/constructor.js
@@ -193,7 +193,7 @@ const Sharp = function (input, options) {
     canvas: 'crop',
     position: 0,
     resizeBackground: [0, 0, 0, 255],
-    useExifOrientation: false,
+    useExifOrientation: true,
     angle: 0,
     rotationAngle: 0,
     rotationBackground: [0, 0, 0, 255],
